### ISSUE TITLE
Displaying global variables or specific address values

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#277]
It'd be great if it's also possible record and display global variables or specific address values at each function entry and exit or event occured.

Example usages are:
- watching a global variable
- watching GOT table changes

We can also apply the same type display method for it.

**Comment by namhyung** [comment]
Looks useful, it can be a read trigger or watch trigger (only shows if differs).

**Comment by honggyukim** [comment]
I agree with you using read trigger.  It's not urgent issue but want to leave a note on the issue list.  You can close this issue and reopen it when you're ready to do this work.  Thanks!

**Comment by honggyukim** [comment]
I think we don't have to show such global data every function entry and exit. Rather than that, we can show them only when they are updated just like gdb watch feature.

**Comment by ParkHanbum** [comment]
there is some issue to use the registers for debugging purpose. 
we can enhance it after implemented. for now, why not implement it to record global variable where each enter/exit of mcount.

**Comment by namhyung** [comment]
We don't (shouldn't?) need to use a hardware register for watch points.  Please check #522 for the implementation.

**Comment by namhyung** [comment]
Please see #661
